extensions/resource-deployment/src/ui/deploymentDialog.ts

```diff
@@ -10,12 +10,14 @@ import * as vscode from 'vscode';
 import { DialogBase } from './dialogBase';
 import { INotebookService } from '../services/notebookService';
 import { DeploymentProvider, DialogFieldInfo, FieldType } from '../interfaces';
+import { isArray } from 'util';
```

Comment:
out of curiosity is this any different from just doing Array.isArray? Does this implementation give us anything as a bonus?

I’m leveraging Array.isArray() in notebooks code, so this would be good for me to understand 😄

Reply:
thanks for catching it, i changed to use Array.isArray. not meant to use this. actually if you go to the definition of this, it tells you to use Array.isArray instead.